src/system_library.jl

```diff
@@ -133,22 +133,6 @@ function build_c_sys5_re(; kwargs...)
         end
     end
 
-    if get(kwargs, :add_single_time_series, false)
```

Comment:
Why are you removing this?

Reply:
Ignore this for now, there are few other changes in here too, once your PR is merged I'll close this and open a new one with the relevant changes

Reply:
For the tests to pass we also need a new IS release